include/z64.h

```diff
@@ -1599,7 +1599,10 @@ struct GlobalContext {
     /* 0x000B0 */ SceneCmd* currentSceneVram;
     /* 0x000B4 */ UNK_TYPE1 padB4[0x4];
     /* 0x000B8 */ View view;
-    /* 0x00220 */ CameraContext cameraCtx;
+    /* 0x00220 */ Camera activeCameras[4];
+    /* 0x00800 */ Camera* cameraPtrs[4];
+    /* 0x00810 */ s16 activeCamera;
+    /* 0x00812 */ s16 unk5F2; //TODO: Name this, was part of CameraContext.
```

Comment:
Rename this to the offset it is at now.
```suggestion
    /* 0x00812 */ s16 unk812;
```